tests/acceptance/blocks/thememap.js

```diff
@@ -29,6 +38,57 @@ class ThemeMap {
   async clickSearchThisAreaButton () {
     await t.click(this._searchThisAreaButton);
   }
+
+  async clickMap () {
+    await t.wait(10000);
```

Comment:
were these 10s waits for testing?

Reply:
ah, my bad. I forget to take these out after testing